uploader.json

```diff
@@ -1939,6 +1939,21 @@
             "pattern": [],
             "symbolic_name": "multipath_conf"
         },
+        {
+            "file": "/var/log/mysql/mysqld.log",
+            "pattern": [],
+            "symbolic_name": "mysql_log"
+        },
+        {
+            "file": "/var/log/mysql.log",
+            "pattern": [],
+            "symbolic_name": "mysql_log"
+        },
+        {
+            "file": "/var/opt/rh/rh-mysql*/log/mysql/mysqld.log",
```

Comment:
@JoySnow this spec won't work in the file section.  If you move this to the glob section of the JSON then it will work correctly.

Reply:
@bfahr  Thank you. I've moved the spec to glob section.